client/my-sites/checkout/upsell-nudge/index.jsx

```diff
@@ -189,14 +190,14 @@ export class UpsellNudge extends React.Component {
 			url
 		);
 	}
-
-	handleClickDecline = () => {
+	handleClickDecline = ( shouldHideUpsellNudges = true ) => {
 		const { trackUpsellButtonClick, upsellType, handleCheckoutCompleteRedirect } = this.props;
 
 		trackUpsellButtonClick( `calypso_${ upsellType.replace( /-/g, '_' ) }_decline_button_click` );
-		handleCheckoutCompleteRedirect();
+		handleCheckoutCompleteRedirect( shouldHideUpsellNudges );
 	};
 
+
```

Comment:
We need to remove this line since it's giving a lint error.

Reply:
Fixed in https://github.com/Automattic/wp-calypso/pull/43199/commits/1b1f72a08cbbf1c177aeb1f49716948b234ed1ee